tests/test_trajectory.py

```diff
@@ -71,7 +71,7 @@ def test_static_integration(example):
     "case",
     [
         (0.0000001, False, False),
-        (1000, True, True),
+        (1000, True, False),
```

Comment:
We now initialize the state with  `has_turned=False`, which is a better default value.